guide/src/SUMMARY.md

```diff
@@ -19,6 +19,9 @@
   - [Conversion traits](conversions/traits.md)]
 - [Python exceptions](exception.md)
 - [Calling Python from Rust](python_from_rust.md)
+- [Using `async` and `await`](async-await.md)
+  - [Awaiting Python awaitables]()
+  - [Other Python async runtimes]()
```

Comment:
I understand why you added these but I'd rather not have the dead links in the guide until we've definitely added support for these.

```suggestion
```

Reply:
They are not deadlinks, they are draft chapters as stated in the mdBook documentation. They are equivalent to the cancellation section with *To be implemented*. And because I've already implemented all of these features, I prefer to showcase the titles (especially `Awaiting Python awaitables`), because I know the PR will come.
I will still remove them as asked. Tell me if I've convinced you to put them back :)